Khan/khan-linter
# Add eslint-plugin-import

Summary:
This eslint plugin has some useful rules.  In particular we'd like to
make use of the "extensions" rule to ensure that all imports have
extensions.  In a recent code review we spotted an import that was
missing its extension.  I'll add this rule to .eslintrc in webapp
in a separate commit.

The files in this PR were generated by running `yarn install eslint-plugin-import`.
Version 2.18.0 of this plugin was installed.

NOTE: This was created as a PR, b/c arc took a long time linting all 
the files and then bail b/c the request size was too big.  Even when
passing `--less-context` to `arc`, it still failed.

Test Plan:
- add `"import/extensions": ["error", "always", {"ignorePackages": true} ],` to .eslintrc in webapp
- check that violations in course-intro-custom-button-options.jsx (in webapp) are flagged

Reviewers: #fe-infra
